docs/build/sdk/indexs.md

```diff
@@ -26,41 +26,62 @@ const res = await sdk.models.getAllMarketIds();
 
 ## createCpmmMarketAndDeployAssets
 
-Create a market using CPMM scoring rule, buy a complete set of the assets used
-and deploy within and deploy an arbitrary amount of those that's greater than
-the minimum amount.
+Creates a market using CPMM scoring rule, buys a complete set of the assets used and deploys the funds.
 
 ```typescript
 const sdk = await SDK.initialize(endpoint);
 
 const res = await sdk.models.createCpmmMarketAndDeployAssets({
-  signer,
-  oracle,
-  period: marketPeriod,
-  marketType,
-  mdm,
-  amount,
-  weights: weights.split(`,`),
-  metadata,
+  signer: util.signerFromSeed(`//Alice`),
+  oracle: `dE3pPiRvdKqPD5bUDBu3Xpi83McE3Zf3UG8CbhWBQfvUywd7U`,
+  period: { block: [4000, 5000] },
+  marketType: { categorical: 5 },
+  metadata: {
+    categories: [
+      { name: `karura` },
+      { name: `moonriver` },
+      { name: `phala` },
+      { name: `robonomics` },
+      { name: `kilt` },
+    ],
+    slug: `kusama-derby-example`,
+    description: `example description`,
+    question: `who will win?`,
+  },
+  mdm: { authorized: `dE3pPiRvdKqPD5bUDBu3Xpi83McE3Zf3UG8CbhWBQfvUywd7U` },
+  swapFee: `10`,
```

Comment:
If you use `10`, then you've a swap fee of `1e-09` percent, which is probably not what you want.

```suggestion
  swapFee: `1_000_000_000`,
```